Commands/rrc.py

```
@@ -171,3 +171,5 @@ def reply_with_random_reddit_comment(self, message):
         print(f"[Error] {e}")
         self.send_privmsg(cmd.channel, f"{cmd.username} Failed to fetch Reddit comment. Please try again later.")
 
+    finally:
+        return
```

Ruff (0.11.9)

`return` inside `finally` blocks cause exceptions to be silenced

(B012)